pkgs/tools/misc/pgmetrics/default.nix

```diff
@@ -1,17 +1,17 @@
-{ lib, buildGoModule, fetchFromGitHub }:
+{ lib, buildGo118Module, fetchFromGitHub }:
 
-buildGoModule rec {
+buildGo118Module rec {
```

Comment:
Please override that in top-level so that it can be dropped more easily when 1.18 becomes the default go version.